### DISCUSSION TITLE
Ensemble of Teacher for ImageNet

### DISCUSSION THREAD

**Discussion by AhmedHussKhalifa** [yoshitomo-matsubara/torchdistill#244] [Q&A] [answered]
Hi @yoshitomo-matsubara ,
Thank you so much for your hard work.

I'm attempting to use your platform to incorporate multiple teachers in order to train one student for imagenet. Could you confirm and clarify the procedures I need to take to add this component in your code [here](https://github.com/yoshitomo-matsubara/torchdistill/blob/08d1f3d7d745e1affda9389496c9c1d40e5e725e/torchdistill/core/distillation.py#L46)?

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @AhmedHussKhalifa 

The easiest way would be to include all the teachers in one `nn.Module` and define it as a teacher model in your yaml file.
If you can provide more details (what models you want to use as part of the ensemble, what your loss function looks like, etc), I can give you a concrete procedure.
